trick_source/sim_services/VariableServer/VariableServerListenThread.cpp

```diff
@@ -27,7 +27,7 @@ Trick::VariableServerListenThread::VariableServerListenThread() :
 
 Trick::VariableServerListenThread::~VariableServerListenThread() {
     if ( listen_dev.hostname != NULL ) {
```

Comment:
Might as well fix this while you're in here. No need to check for null pointers before freeing.
https://stackoverflow.com/a/6084233

Reply:
Got it.